### ISSUE TITLE
"setAlias()" and "unregister();"doesn't work with compiled APK.

### ISSUE THREAD

**Issue by joanebrown** [pushbots/phonegap#54]

**Comment by AbdullahDiaa** [comment]
You should use this method instead of unregister:
`window.plugins.PushbotsPlugin.toggleNotifications(false);`

Regarding setAlias, can you please share a copy of the log once you call setAlias method, also which version of the library do you use?

**Comment by joanebrown** [comment]
@AbdullahDiaa I will try to use `toggleNotifications`, thanks.

Regarding `setAlias` now I using this on my server side instead of the app, but I will try to give you a log :-)

BTW, can I use one alias for multiple devices?? (for users who have multiple devices)


Regards,

**Comment by joanebrown** [comment]
And why I should use `toggleNotifications(false)` instead of `unregister()`? I use `unregister()` when my users logout of my app.

**Comment by joanebrown** [comment]
Definitely I think there is something wrong with this plugin:

**Steps I perform:**

**1:** In the devices list of pushbots.com there is no device. 

**2:** When I login successfully into my app, I use `initialize()`:

    window.plugins.PushbotsPlugin.initialize(MY_APPLICATION_ID, {
        "android": {
            "sender_id": MY_SENDER_ID
        }
    });
	
**3:** Then, I register `received` and `clicked` (*I don't think it affects the problem*)

    window.plugins.PushbotsPlugin.on("notification:received", function(data) {
        console.info(JSON.stringify(data));
    });

    window.plugins.PushbotsPlugin.on("notification:clicked", function(data) {
        console.info(JSON.stringify(data));
    });

**4:**  Then I register `registered` event (*I use exactly the same inside `getRegistrationId`*):

    console.log("registered???");
    window.plugins.PushbotsPlugin.on("registered", function(token) {
        console.log("registered!!!");

        // shows correctly the username
        console.log(userName); 

        // set alias for the device (my username).
        window.plugins.PushbotsPlugin.setAlias(userName);


    });

**5:** When my user logout (*so I don't want to receive notifications anymore until the user login again, if he does)* I use:

    window.plugins.PushbotsPlugin.unregister();

---

List of possible bugs:

1. `setAlias()` doesn't work when I used it in compiled APK. 

2. `unregister()` doesn't work when I used it in compiled APK

3. When I delete one device from https://pushbots.com and restart the app and then I uninstall and reinstall the app (*or close and open*) `registered` doesn't work.

---
Note: I don't know how to access to log when I have an APK. I only know how to use Ionic log on livereload.

Note2: I tried with `toggleNotifications(false)` instead of `unregister()` but doesn't work. I receive notifications ever...
---

If I'm doing something wrong. How should I perform the procedure described above? I'm desperate..

I use the latest version 1.4.1
